packages/contracts/package.json

```diff
@@ -16,22 +16,24 @@
   "dependencies": {
     "@types/command-line-args": "5.2.0",
     "command-line-args": "5.2.1",
-    "dotenv": "^16.0.0",
+    "dotenv": "^16.0.3",
     "ethers": "^5.6.9",
-    "ts-node": "10.9.1"
+    "tsx": "^3.12.2"
   },
   "scripts": {
     "forge": "forge",
     "build": "forge build",
-    "test": "ts-node ./scripts/runner/runner.ts",
-    "slither:run": "slither . --compile-force-framework foundry",
+    "test": "tsx ./scripts/runner/runner.ts",
+    "slither:run": "slither . --compile-force-framework foundry --config-file slither.config.json",
     "echidna": "echidna-test . --config echidna.config.yml",
     "prettier": "forge fmt",
```

Comment:
I just realized that this could be causing the discrepancy with the formatting. Because this task is named "prettier" it is possible that lint-staged is calling this instead of what we expect.
```suggestion
```

Reply:
if we want to use prettier then we need to update that otherwise it will format based on the settings in the foundry.toml and not inside the .prettierrc

Reply:
I just created a new issue because it looks like @hashedMae wants to leave this as is in this PR.